File: app/helpers/js/bot/bot.js
const puppeteer = require("puppeteer", { headless: true });
const fs = require("fs");

const pages = ["https://www.google.com", "https://www.facebook.com", "https://www.youtube.com"];

(async () => {
  // Inicializa el navegador.
  const browser = await puppeteer.launch();

  // Itera sobre la lista de páginas.
  for (const page of pages) {
    // Crea una nueva página.
    const page = await browser.newPage();

    // Carga la página.
    await page.goto(page);

    // Obtén el título de la página.
    const title = await page.title();

    // Imprime el título.
    console.log(title);
  }

  // Cierra el navegador.
  await browser.close();
})();

File: app/helpers/js/bot/bot.py
import requests

def get_title(url):
  """Obtiene el título de una página web."""
  response = requests.get(url)
  return response.headers["title"]

def main():
  # Define la lista de páginas.
  pages = ["https://www.google.com", "https://www.facebook.com", "https://www.youtube.com"]

  # Itera sobre la lista de páginas.
  for page in pages:
    print(get_title(page))

if __name__ == "__main__":
  main()